### PR TITLE
Removed unnecessary assets and imports.

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #57

### CODE REVIEW

**Comment by Brijeshthummar02** [comment]
Share feedback for current pull request, and if it's fine than please do me a favor merge it in oct 1-2 when hacktoberfest starts so that i can claim merge request.

File: src/assets/cash_flow.png
Comment (mittal-parth) [inline src/assets/cash_flow.png:1]: Don't remove this

File: src/assets/logo.svg
Comment (mittal-parth) [inline src/assets/logo.svg:1]: Don't think this should be removed either.

**Review by mittal-parth — CHANGES_REQUESTED** [review]
You also have to remove their imports from [src/assets/index](https://github.com/mittal-parth/personal-portfolio/blob/main/src/assets/index.js) and thereby from [constants/index](https://github.com/mittal-parth/personal-portfolio/blob/main/src/constants/index.js)

The current ones that you have removed are not exhaustive. There are more to be removed. The best way to identify that is to check the two files I mentioned above. 

If the file exists in assets but is not imported in assets/index - remove it
If the file is imported but is then not used in constants/index - remove it

File: src/assets/logo.svg
Comment (Brijeshthummar02) [reply src/assets/logo.svg:1]: okayy so i'm changing everything and creating a new pr soon

Comment (mittal-parth) [reply src/assets/logo.svg:1]: Don't make a new PR. You can do the changes here on the same branch. In any open source repo, you might comments on the PR, after multiple iterations also. That would be inconvenient if every comment results in a new PR. So you can continue on the same one.

**Pull request closed without merging**